.github/workflows/test-pull-request.yaml

```diff
@@ -26,10 +26,34 @@ jobs:
         with:
           node-version: "16.x"
 
-      # install deps for frontend and library and build both
-      - name: Install library and frontend deps and build
+      # install deps for Library development and build
+      - name: Install library for development
+        working-directory: ./library
+        id: npm
+        run: |
+          npm ci
+          npm run build-dev
+
+      # install deps for Library production and build
+      - name: Install library for production
+        working-directory: ./library
+        id: npm
+        run: |
+          npm ci
+          npm run build-prod
+
+      # install deps for Frontend and Library for development and build both
+      - name: Install library and Frontend for development
+        working-directory: ./examples/typescript
+        id: npm
+        run: |
+          npm ci
+          npm run build-all-dev
+      
+      # install deps for Frontend and Library for production and build both
+      - name: Install library and Frontend for production
         working-directory: ./examples/typescript
         id: npm
```

Comment:
Remove the id here we dont need it

Reply:
resolved in 57089f65c30a4d20bdbe5aae074acd237aa69b81